aurget

```diff
@@ -193,16 +195,18 @@ buildpkg() {
 
   if $silent; then
     debug 'building silently...'
+    # TODO: SC2086: Double quote to prevent globbing and word splitting.
     $makepkg $makepkg_options "$@" --noconfirm &>/dev/null
```

Comment:
I think we expect `makepkg` to always be a single word, so `"$makepkg"` should work here. I don't even know why we have this as configurable to be honest. Doe anyway use some alternative `makepkg`?

The `makepkg_options` variable should definitely be made an array, IMO. I can't remember if there's complexity around why it's not.